azure/functions/decorators/function_app.py

```diff
@@ -1326,140 +1463,104 @@ def decorator():
 
         return wrap
 
-    def generic_trigger(self,
-                        arg_name: str,
-                        type: str,
-                        data_type: Optional[Union[DataType, str]] = None,
-                        **kwargs
-                        ) -> Callable:
+
+class FunctionRegister(DecoratorApi, HttpFunctionsAuthLevelMixin, ABC):
```

Comment:
So - this would be used by the Worker now?

/cc @gavin-aguiar

Reply:
yes, need to make change on worker to index FunctionRegister instead of functionapp